Accessories/uInverter/Design.md

```diff
@@ -0,0 +1,53 @@
+﻿## uInverter Design Calculations
+
+
+-   nominal dc link voltage - _12V_
+-   rated phase current - _2.5A rms_
+-   range of allowable switching frequency - _1kHz- 100kHz_
+-   range of allowable power frequency (phase frequency) - _640- 1000 Hz_
+
+ 
+### Core Components
+ #### DC Barrel Jack
+ 
+ The 2.1mm [DC barrel jack](https://www.digikey.com/en/products/detail/3643/1528-2553-ND/8346574) is rated for 15Vdc and is supplied from a standard [AC-DC wall adapter](digikey.com/en/products/detail/cui-inc/SWM30-12-NV-P5/7388285) rated for 12V. 
+
+#### DC link 
+A 3mF capacitor is selected as a DC link capacitor to protect the voltage ripple from input to the components comprising the half bridge. The capacitor is should be able to withstand a 12V input voltage. 
```

Comment:
I think this sentence should go away since the below discussion computes a new value based on an equation? If not, where did 3mF come from?